src/calculation/HTMLBuilder.ts

```diff
@@ -243,19 +243,11 @@ export function generateClauseCoverageHTML(
     const flattenedStatementResults = detailedResults.flatMap(s => s.statementResults);
     const flattenedClauseResults = detailedResults.flatMap(c => (c.clauseResults ? c.clauseResults : []));
 
-    // Grab every statement with any relevance other than N/A
-    // There may be multiple entries for a given statement across the results,
-    // but we know that non of them can be irrelevant
-    const relevantStatements = uniqWith(
-      flattenedStatementResults,
-      (s1, s2) => s1.libraryName === s2.libraryName && s1.localId === s2.localId && s1.relevance === s2.relevance
-    ).filter(s => s.relevance !== Relevance.NA);
-
-    // From all the relevant ones, filter out any duplicate statements
+    // Filter out any statement results where the statement relevance is not NA and filter out duplicates from that
```

Comment:
```suggestion
    // Filter out any statement results where the statement relevance is NA and filter out duplicates from that
```

We are filtering out the `NA`s keeping everything else.